rootex/core/renderer/post_processor.h

```diff
@@ -52,16 +54,38 @@ class PostProcess
 	virtual void draw(CameraComponent* camera, ID3D11ShaderResourceView*& nextSource) = 0;
 };
 
-class PostProcessor
+class CustomPostProcess : public PostProcess
 {
-	Vector<Ptr<PostProcess>> m_PostProcesses;
+	String m_PostProcessPath;
+
+	EventBinder<CustomPostProcess> m_Binder;
+
+	BufferFormat bufferFormat;
```

Comment:
since this can be shred between the various CustomPostProcesses, make it static, rename to s_BufferFormat, and add a check similar to s_VertexShader as suggested above